package.json

```
@@ -139,6 +139,6 @@
   ],
   "license": "MIT",
   "engines": {
-    "node": "^10.12.0 || >=12.0.0"
+    "node": "^12.22.0 || ^14.17.0 || >=15.3.0"
```

Node.js v15 is EOL, so I think we should start at v16 instead

```suggestion
    "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
```

---

make sense to me! https://github.com/nodejs/Release/pull/674

cc @eslint/eslint-tsc

---

If we were about to release v8.0.0, I'd probably lean toward supporting Node v15 with the EOL date only four days ago, but since we haven't even started prereleases yet, that should give us a couple months' padding. Removing Node v15 support for a release still a couple months away makes sense to me too :+1:

---

dropping Node v15 does not mean users cannot run eslint on it. Instead, I think it can run without any issues. 😄

---

Makes sense to me.